Library/ViewModels/RemoteConfigFeatureFlagToolsViewModelTests.swift

```diff
@@ -45,14 +41,11 @@ final class RemoteConfigFlagToolsViewModelTests: TestCase {
 
   func testReloadWithData_FeaturesEnabledAndDisabled() {
     let mockRemoteConfigClient = MockRemoteConfigClient()
-      |> \.features .~ [
-        RemoteConfigFeature.blockUsersEnabled.rawValue: false,
-        RemoteConfigFeature.consentManagementDialogEnabled.rawValue: true,
-        RemoteConfigFeature.darkModeEnabled.rawValue: false,
-        RemoteConfigFeature.facebookLoginInterstitialEnabled.rawValue: true,
-        RemoteConfigFeature.postCampaignPledgeEnabled.rawValue: false,
-        RemoteConfigFeature.reportThisProjectEnabled.rawValue: false
-      ]
+
+    for feature in RemoteConfigFeature.allCases {
+      let trueOrFalse = feature.rawValue.hashValue % 2 == 1
```

Comment:
This is so each flag will get a stable true or false value, but we don't have to keep adding them to this list.